util/ifutil/ifutil.go

```diff
@@ -130,3 +130,18 @@ func Diff(labelA string, a interface{}, labelB string, b interface{}) string {
 	}
 	return diff[:len(diff)-2]
 }
+
+// Must takes an object and an error, and panics if the error is not nil. It can be used for
+// inlining function calls when you know that the function won't error.
+func Must[T any](value T, err error) T {
+	if err != nil {
```

Comment:
Without the suggested change, the function will panic in the unit test below.

```suggestion
	if !IsNil(err) {
```

unit-test

```
// verr implements error
type verr struct{}

func (e *verr) Error() string { return "" }

func TestMust(t *testing.T) {
	validate := func() *verr {
		return nil
	}

	x := Must(1, validate())
	require.Equal(t, 1, x)
}
```

Reply:
Nice catch! Thanks Gilles.